lib/load-simple-icons.js

```diff
@@ -3,9 +3,17 @@
 const simpleIcons = require('simple-icons')
 const { svg2base64 } = require('./svg-helpers')
 
+const logoBlacklist = ['travis']
```

Comment:
The simple-icon logo is `travis-ci`, right?

Reply:
sorry had already resolved this, just had to push the change :smile: